static/app/utils/theme.tsx

```diff
@@ -414,35 +233,126 @@ const generateAliases = (colors: BaseColors) => ({
   bannerBackground: colors.gray500,
 });
 
-const dataCategory = {
-  [DATA_CATEGORY_INFO.error.plural]: CHART_PALETTE[4][3],
-  [DATA_CATEGORY_INFO.transaction.plural]: CHART_PALETTE[4][2],
-  [DATA_CATEGORY_INFO.attachment.plural]: CHART_PALETTE[4][1],
-  [DATA_CATEGORY_INFO.replay.plural]: CHART_PALETTE[4][4],
-  [DATA_CATEGORY_INFO.monitorSeat.plural]: '#a397f7',
+type Alert = 'muted' | 'info' | 'warning' | 'success' | 'error';
+
+type AlertColors = {
+  [key in Alert]: {
+    background: string;
+    backgroundLight: string;
+    border: string;
+    borderHover: string;
+    color: string;
+    // @TODO(jonasbadalic): Why is textLight optional and only set on error?
+    textLight?: string;
+  };
 };
 
-/**
- * Default colors for data usage outcomes
- */
-const outcome = {
-  [Outcome.ACCEPTED]: CHART_PALETTE[5][0], // #444674 - chart 100
-  [Outcome.FILTERED]: CHART_PALETTE[5][2], // #B85586 - chart 300
-  [Outcome.RATE_LIMITED]: CHART_PALETTE[5][3], // #E9626E - chart 400
-  [Outcome.INVALID]: CHART_PALETTE[5][4], // #F58C46 - chart 500
-  [Outcome.CLIENT_DISCARD]: CHART_PALETTE[5][5], // #F2B712 - chart 600
-  [Outcome.DROPPED]: CHART_PALETTE[5][3], // #F58C46 - chart 500
-};
+const generateUtils = (colors: Colors, aliases: Aliases) => ({
```

Comment:
Thoughts on moving `generate*` utils to a different file?

Reply:
I want to delete these tbh, and I think I can do that if I create a single generateTheme type function that will be able to generate both light and dark, then I can just inline it, it would just be a lot more to do